setup.py

```diff
@@ -0,0 +1,74 @@
+import os
+from builtins import *
```

Comment:
avoid importing `*`
```suggestion
import builtins
```
or
```suggestion
from builtins import [...]
```